src/site/content/en/blog/squoosh-v2/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Announcing Squoosh v2
+subhead: New codecs support, updatd design, and CLI support!
+description: |
+  Major version change to Squoosh.app with new CLI feature.
+scheduled: true
+date: 2020-12-09
+hero: hero.jpg
+alt: A screeen capture of new Squoosh top page
+tags:
+  - blog
+  - tools
+  - javascript
+  - images
+---
+
+
+
+[Squoosh](https://squoosh.app) is an image compression app our team built and [debuted at Chrome Dev Summit 2018](https://youtu.be/ipNW6lJHVEs). We built it to make it easy to experiment with different image codecs, and to showcase the capabilities of the modern web.  
+
+Today, we are releasing a major update to the app with more codecs support, a new design, and a new way to use Squoosh on your command line called Squoosh CLI.
+
+
+## New codecs support
+
+We now support OxiPNG, MozJPEG, WebP, and AVIF, in addition to codecs natively supported in your Browser.  A new codec was made possible again with the use of WebAssembly. By compiling a codec encoder and decoder as WebAssembly module users can access and experiment with newer codecs even if their preferred browser does not support them. 
```

Comment:
```suggestion
We now support OxiPNG, MozJPEG, WebP, and AVIF, in addition to codecs with built-in support in your browser.  A new codec was made possible again with the use of WebAssembly. By compiling a codec encoder and decoder as a WebAssembly module users can access and experiment with newer codecs even if their preferred browser does not support them. 
```
"native" is on P&E's "do not use" word list because it has some bad political assocations